### DISCUSSION TITLE
Approval of WalletConnect registry.

### DISCUSSION THREAD

**Discussion by ReGen71** [WalletConnect/walletconnect-monorepo#944] [Developer Support]
Hi there!
We have submitted an application to register our wallet "TTM Wallet"
Would you be so kind as to approve the application so that our users can access WallectConnect as soon as possible.
Thanks in advance

**Comment by kumarnagender09** [comment]
They are not replying

**Reply by chadyj** [reply]
Ser, the listing was approved, and is now online https://walletconnect.com/registry?search=ttm
